packages/web-app-files/src/views/shares/SharedWithMe.vue

```diff
@@ -166,9 +182,28 @@ export default defineComponent({
         )
       }
 
+      if (unref(filterTerm).trim()) {
+        const usersSearchEngine = new Fuse(result, { ...defaultFuseOptions, keys: ['name'] })
+        result = usersSearchEngine.search(unref(filterTerm)).map((r) => r.item)
```

Comment:
The search order is apparently unstable like this, if that is desired you need to make it a filter on the original items, see #10046